cdc_rsync/cdc_rsync.cc

```diff
@@ -69,7 +70,8 @@ PathFilter::Rule::Type ToInternalType(FilterRule::Type type) {
 ReturnCode Sync(const Options* options, const FilterRule* filter_rules,
                 size_t num_filter_rules, const char* sources_dir,
                 const char* const* sources, size_t num_sources,
-                const char* destination, const char** error_message) {
+                const char* user_host, const char* destination,
```

Comment:
I'm not 100% clear yet how we'lll finally do it. It would make sense to just take a target param that includes user@host:path (or even a URI). This would make sense if we eventually support local copies, but for now let's keep it this way.